python/pyarrow/tests/test_pandas.py

```diff
@@ -1044,7 +1044,7 @@ def test_python_datetime_with_pytz_tzinfo(self):
             df = pd.DataFrame({'datetime': values})
             _check_pandas_roundtrip(df)
 
-    @h.given(st.none() | tzst.timezones())
+    @h.given(st.none() | st.timezones())
```

Comment:
Can't remember if this is the correct change deleted with rebasing or not :(

Reply:
Or this can be `past.timezones()` instead?